max_lin_vel: 0.6
max_ang_vel: 1
cmd_vel_topic: turtlebot/cmd_vel
